packages/grafana-ui/src/components/Table/FooterRow.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { ReactNode } from 'react';
+import { ColumnInstance, HeaderGroup } from 'react-table';
+import { selectors } from '@grafana/e2e-selectors';
+import { getTableStyles, TableStyles } from './styles';
+import { useStyles2 } from '../../themes';
+
+export interface FooterRowProps {
+  totalColumnsWidth: number;
+  footerGroups: HeaderGroup[];
+  footer?: ReactNode[];
+  height?: number;
+}
+
+export const FooterRow = (props: FooterRowProps) => {
+  const { totalColumnsWidth, footerGroups, footer, height } = props;
+  const e2eSelectorsTable = selectors.components.Panels.Visualization.Table;
+  const tableStyles = useStyles2(getTableStyles);
+
+  if (!footer) {
+    return null;
+  }
+
+  return (
+    <table
+      style={{
+        position: 'absolute',
+        width: totalColumnsWidth ? `${totalColumnsWidth}px` : '100%',
+        bottom: '0px',
+      }}
+    >
+      {footerGroups.map((footerGroup: HeaderGroup) => {
+        const { key, ...footerGroupProps } = footerGroup.getFooterGroupProps();
+        return (
+          <tfoot
+            className={tableStyles.tfoot}
+            {...footerGroupProps}
+            key={key}
+            aria-label={e2eSelectorsTable.footer}
```

Comment:
think we are not supposed to use aria-labels for this anymore (only used for accessasbility reasons)